packages/website/src/theme/MDXComponents/TryInPlayground.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import styles from './TryInPlayground.module.css';
+
+export function TryInPlayground({
+  eslintrcHash,
+}: {
+  eslintrcHash: string;
+}): React.ReactNode {
+  return (
+    <div>
+      <a
+        href={`/play#eslintrc=${eslintrcHash}`}
+        className={styles.tryInPlaygroundLink}
```

Comment:
I'm remembering now that folks generally associate the ↗ with opening in a new tab.

```suggestion
        className={styles.tryInPlaygroundLink}
        target="_blank"
```